lib/controller.js

```diff
@@ -7,7 +7,10 @@ var sprintf = require('sprintf-js').sprintf;
 var cp = require('child_process');
 var async = require('async');
 var updates = require('./update-fetch');
+var debug = require('debug')('controller');
+var Menu = require('terminal-menu');
 var controller = {};
+controller.TESSEL_AUTH_KEY = Tessel.TESSEL_AUTH_PATH + '/id_rsa';
```

Comment:
I think TESSEL_AUTH_KEY and TESSEL_AUTH_PATH should be set in the same place. For someone looking to maintain this code in the future, it would be confusing that one is set in `controller.js` and the other is set in `provision.js`. 

@Student007 I think the better option would be to not set the default in `tessel-2.js` and just substitute the standard Tessel key path in `controller.js`. 

I'm always partially thinking about how we can make this CLI eventually usable for other types of devices and to do that we would need to separate the device key location from the command parsing.